src/Browser/BrowserProcess.php

```diff
@@ -111,16 +111,13 @@ public function start($binaries, $options)
         $this->logger->debug('process: using directory: ' . $options['userDataDir']);
 
         // get args for command line
-        $args = $this->getArgsFromOptions($options);
-
-        // process string
-        $processString = $binaries . ' ' . implode(' ', $args);
+        $args = $this->getArgsFromOptions($binary, $options);
 
         // log
-        $this->logger->debug('process: starting process: ' . $processString);
+        $this->logger->debug('process: starting process: ' . implode(' ', $args));
```

Comment:
This should be
```php
$this->logger->debug('process: starting process: ' . $process->getCommandLine());
```
and be put after `$process` has been initiated before calling `$process->start()`.